internal/pkg/service/buffer/store/model/column/column_test.go

```diff
@@ -171,5 +173,5 @@ func TestColumn_Template_UndefinedKeyNil(t *testing.T) {
 
 	val, err := c.CsvValue(column.ImportCtx{Body: body, Header: header})
 	assert.NoError(t, err)
-	assert.Equal(t, "", val)
+	assert.Equal(t, "null\n", val)
```

Comment:
neviem ci by `null` nemala byt v CSV ako prazdny string.
Ak pride string null, tak to bude `"null"`? 

Neviem ako mas vyriesene tie datove typy, ... ci sa berie ze ta hodnota je vzdy JSON (potom je `null` spravne, kedze string by bol `"null"`).

Tj. len poznamka, ze sa treba nad tym zamysliet a vysvetlit to potom v dokumentacii.

Reply:
Ok. Ten `Template.DataType` jsem neřešil. Někam jsem psal (nebo jsem aspoň myslel že jsem psal 😄), že moc nechápu k čemu to je, resp. že to tady asi nemá smysl řešit, když se všechno zapíše do CSV jako string. Nebo ne?

Reply:
Ok (nevidel som to zatial asi nikde), to sme mali diskuovat skor pri priprave design dokumentu, ale mozeme aj teraz.

Preberieme na standupe, zaver moze byt ze to zrusime a vysledok je vzdy JSON string (nie som proti).
Len to treba vyhodit z `design.go` a skrtnut v design doc.

Reply:
No nevím, to bych ten dokument musel studovat tak týden, abych si dopředu uvědomil všechny souvislosti. 😅 A ani jsem moc neměl představu jak se ten Jsonnet mapping bude používat, tak jsem netušil kde se má ten data type použít. 🤷‍♂️

Reply:
Ok, to si mal vtedy tusim akurat dovolenku, ked sa to riesilo.

Tym som chcel len povedat, ze by som privital sirsiu diskusiu, ked pripravujeme design doc a neviem ako to dosiahnut :slightly_smiling_face:

Reply:
To myslíš kdy, po tom co jsem se vrátil? Design doc byl z 80% úplně přepsaný a vy jste zřejmě měli dobrou představu jak to má fungovat. Tak jsem to měl celý postupně procházet a řádek po řádku komentovat a rozporovat? Jako klidně, ale musíme si to říct. Samozřejmě jsem to po návratu četl a jo, některý části mi nebyly jasný, ale v design doc není detailní implementační příručka, čekal bych takovýhle detaily spíš v popisu jira issues.

Reply:
> neviem ci by null nemala byt v CSV ako prazdny string.

V tom případě ale musím vrátit `"\n"` aby to bylo konzistentní.

Reply:
> V tom případě ale musím vrátit "\n" aby to bylo konzistentní.

@JakubMatejka sorry, nedoslo mi ze je to hodnota `column` nie row.
Z column by som to `\n` na konci radsej odstranil, aby s tym nejaky JSON parser nemal problem.

Reply:
jasně, ok

Reply:
🤔 to bych měl ale taky oddělat ty uvozovky odtud: https://github.com/keboola/keboola-as-code/blob/84f21dbd2be7288bc392e424a567fd1de806f31f/internal/pkg/service/buffer/store/model/column/column_test.go#L131-L131

Reply:
no nie, iba to `'n` (?)

Hodnota je: `"val2"`, tj. JSON encoded string

Potobne `"null"` je JSON encoded string.
A `null` je null.

Ak neriesime data typy a nejake konverzie, tj. vystup je vzdy JSON, tak mi to pride spravne, tebe nie?

Reply:
No je to hodnota která se zapíše do csv buňky. Ale v našem případě je uvozovka csv enclosure, tak to bude jedno že se tam připíše.

Reply:
@JakubMatejka ale CSV writer predsa este prida jedny `"` ci nie?

Reply:
A tie existujuce escapne

Reply:
Takze CSV rov bude `.....,"""val2""",...\n`

Reply:
(ak ta myli nazov metody `CsvValue` tak to premenuj na `ColumnValue`)

Reply:
> Takze CSV row bude `.....,"""val2""",...\n`

No jo vlastně. Tomu rozumím, jen si nejsem jistý že to chceme. 😅

Reply:
No to mali riesit datove typy, .... ak stripnes `"` tak uz nerozoznas `"null"` a `null` -> ze to musis potom spravit nejak konfigurovatelne, ..... alebo previes `null` -> empty string (ale to zase nerozlisis empty string a null)

Reply:
No v csv, kam se ten výsledek zapíše, empty string a null nerozeznáš.

Reply:
Nechajme diskusiu na standup o 2 min :slightly_smiling_face: